app/models/host.rb

```diff
@@ -21,6 +21,7 @@ class Host < ApplicationRecord
     "redhat"          => "RedHat",
     "vmware"          => "VMware",
     "openstack_infra" => "OpenStack Infrastructure",
+    "oracle"          => "Oracle",
```

Comment:
@matthewd @durandom @juliancheal this needs to be pushed back into each class.  If needed, it can be aggregated here.